internal/handlers/tigris/msg_count.go

```diff
@@ -16,13 +16,110 @@ package tigris
 
 import (
 	"context"
+	"fmt"
 
+	"github.com/FerretDB/FerretDB/internal/handlers/common"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
 	"github.com/FerretDB/FerretDB/internal/util/must"
 	"github.com/FerretDB/FerretDB/internal/wire"
 )
 
 // MsgCount implements HandlerInterface.
 func (h *Handler) MsgCount(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
-	// TODO https://github.com/FerretDB/FerretDB/issues/771
-	return nil, notImplemented(must.NotFail(msg.Document()).Command())
+	document, err := msg.Document()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	unimplementedFields := []string{
+		"skip",
+		"returnKey",
+		"showRecordId",
+		"tailable",
+		"oplogReplay",
+		"noCursorTimeout",
+		"awaitData",
+		"allowPartialResults",
+		"collation",
+		"allowDiskUse",
+		"let",
```

Comment:
could you please clarify where did you take all that parameter list bcz I don't see them in `count` unimplemented fields for Postgres backend and also I don't see it in the issue description and in the `count` description for MongoDB.

Reply:
They are the same as for Tigris' `MsgFind`: https://github.com/FerretDB/FerretDB/blob/main/internal/handlers/tigris/msg_find.go#L35. From the first sight, it looks reasonable to have the same list, but after your comment I'm not sure anymore :)

Reply:
`count` command has different arguments: https://www.mongodb.com/docs/manual/reference/command/count/